src/components/DashHeader.tsx

```diff
@@ -62,8 +63,11 @@ function DashHeader() {
                   alt="logoWhite"
                 />
               )}
+              <h1 className="text-3xl font-bold font-lexend text-primary dark:text-dark-text-fill">
+                PULSE
+              </h1>
             </Link>
-            <div className="relative">
+            {/* <div className="relative">
```

Comment:
It seems you commented this out instead of removing it?

Reply:
Solved.